addons/bus/controllers/websocket.py

```diff
@@ -58,7 +58,7 @@ def update_bus_presence(self, inactivity_period, im_status_ids_by_model):
         return {}
 
     @route('/bus/websocket_worker_bundle', type='http', auth='public', cors='*')
-    def get_websocket_worker_bundle(self):
+    def get_websocket_worker_bundle(self, **kw):
```

Comment:
With the corresponding pylint ignore unused params
```suggestion
    def get_websocket_worker_bundle(self, v=None):
```

So you keep benefiting from the "warning endpoint called with unknown ignored arguments: ..." plus it makes it clear what are the "extra" frontend-only/cache-busting arguments in use and you can document them

Reply:
I don't have the time *right now* to do a deeper review